components/core/cmake/Toolchains/llvm-clang-toolchains.cmake

```
@@ -0,0 +1,13 @@
+message(STATUS "Setting up LLVM v16 toolchains for MacOS 13 Ventura and MacOS 14 Sonoma...")
+
+execute_process(
+    COMMAND
+        brew --prefix llvm@16
```

```suggestion
        "brew" "--prefix" "llvm@16"
```

Let's quote strings that aren't parameter or variable names to make the scripts more readable.